local-dev-environment/testing/evaluation_config.py

```diff
@@ -0,0 +1,119 @@
+"""Configuration management for relevance scoring.
+
+This module handles search configuration and determines the appropriate
+term checking method based on active search types.
+
+Run from local-dev-environment directory: python -m testing.run_evaluation
+"""
+
+from datetime import datetime
+from pathlib import Path
+
+from testing.evaluation_settings import (
+    ANALYSER_BOOST,
+    FUZZINESS,
+    FUZZY_BOOST,
+    K_QUERIES,
+    KEYWORD_BOOST,
+    MAX_EXPANSIONS,
+    SCORE_FILTER,
+    SEMANTIC_BOOST,
+    WILDCARD_BOOST,
+)
+
+# Output directories
```

Comment:
Output directories? 
This one is an input? 
CHUNKS_FILE = SCRIPT_DIR / "testing_docs" / "TC19_test_all_chunks.csv"